### PR TITLE
automated spellcheck  for md files on pr and commit

### PR DESCRIPTION
**Pull request by mgheorghe**
github action to spellcheck the md files if they get modified.
verified to pass on current azure dash main branch

### CODE REVIEW

**Comment by mgheorghe** [comment]
PR https://github.com/Azure/DASH/pull/189 fixes the spelling errors , that should be reviewed before this, otherwise this check will keep failing

File: .wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,572 @@
...
+BGP
+bgp
+bm
+BMV
+bmv
+BMv
+br
+breakpoint
+Bringup
+bruh
```
Comment (chrispsommers) [inline .wordlist.txt:52]: :)

**Comment by chrispsommers** [comment]
At community WG meeting, there was a broad consensus to enable spell-checking during CI. People making direct commits or submitting PRs will have to correct their spelling and/or add to the `.wordlist.txt` file to pass the check.

@mgheorghe please make adjustments to get a passing CI and we can merge. Thanks!

**Comment by chrispsommers** [comment]
Merging per consensus at today's and last week's DASH WG meeting.

**Pull request merged**